Tests using Mockito fail in IntelliJ

Tests are using Mockito currently fail when ran in IntelliJ IDEA (latest: 2021.1.2).

```
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make protected final java.lang.Class java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain) throws java.lang.ClassFormatError accessible: module java.base does not "opens java.lang" to unnamed module @4b85612c
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
	at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:199)
	at java.base/java.lang.reflect.Method.setAccessible(Method.java:193)
	at org.mockito.cglib.core.ReflectUtils$2.run(ReflectUtils.java:69)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:312)
	at org.mockito.cglib.core.ReflectUtils.<clinit>(ReflectUtils.java:59)
	... 57 more
```

Workaround is to use JDK 15 instead of JDK 16.

--- a/build-tools-internal/src/main/groovy/elasticsearch.ide.gradle
+++ b/build-tools-internal/src/main/groovy/elasticsearch.ide.gradle
@@ -128,7 +128,8 @@ if (System.getProperty('idea.active') == 'true') {
               '--add-opens=java.base/java.net=ALL-UNNAMED',
               '--add-opens=java.base/javax.net.ssl=ALL-UNNAMED',
               '--add-opens=java.base/java.nio.file=ALL-UNNAMED',
-              '--add-opens=java.base/java.time=ALL-UNNAMED'
+              '--add-opens=java.base/java.time=ALL-UNNAMED',
+              '--add-opens=java.base/java.lang=ALL-UNNAMED'
             ].join(' ')
           }
         }

